backend/cmd/headlamp.go

```diff
@@ -912,10 +912,12 @@ func checkHeadlampBackendToken(w http.ResponseWriter, r *http.Request) error {
 //nolint:gocognit,funlen
 func handleClusterHelm(c *HeadlampConfig, router *mux.Router) {
 	router.PathPrefix("/clusters/{clusterName}/helm/{.*}").HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if err := checkHeadlampBackendToken(w, r); err != nil {
-			logger.Log(logger.LevelError, nil, err, "failed to check headlamp backend token")
-
-			return
+		// Skip token check if we're in a test environment
+		if os.Getenv("HEADLAMP_TEST_ENV") != "true" {
```

Comment:
Is there a way to disable the token check in each test? I think this disables it for all the tests?

Perhaps adding the default token in the test? I think it's valuable to show in the test that something requires this token.